api/classes/publisher.js

```diff
@@ -127,262 +120,276 @@ function deleteFileRemotely(file, root) {
   });
 }
 
-function modifyPublishedFiles(action, project, user, files, remixMetadata) {
-  return Promise.resolve()
-  .then(function() {
-    if (files.length) {
-      return files.mapThen(function(file) {
-        return action(file, '/' + user.get('name') + '/' + project.get('id'), remixMetadata);
-      });
+
+/**
+ * Record fetching helpers
+ */
+function fetchProjectModel(id) {
+  return Projects.query({
```

Comment:
I'll add a comment above but this is the only method that does it since we accept a Bookshelf model in our entry point and we want to send back a Bookshelf model; this method allows us to do that. the `Queries` interface was done to return regular javascript objects since we tap directly into knex